docs/common-arguments.md

```diff
@@ -1,132 +1,118 @@
 # Common arguments
-## Argument ```--project-select {project_id}```
+## Argument ```--output```
+Specify output format of command.
 
-A parameter intended for selecting an ad-hoc project for a single command.
-
-###Examples
-
-#### List virtual machine on selected project
+#### Available formats
 
-```bash
-{{scope}} vm list --output tsv --project-select 5b34c27bddd5b3c5e9fbaab0
-```
-## Argument ```--output table / -o table```
+* ```table``` - Simple results as tabular (default).
+* ```list``` - Subsequent properties line by line.
+* ```tsv``` - Mostly works like tabular format, but without the header row. Efficiently usable in ```awk``` and shell.
+* ```json``` - Raw response of API
+* ```yaml``` - Full response displayed as YAML
+* ```id``` - Just resource ID line by line
 
-### Description
+##### Examples
 
-Default output format, which can be overwritten by the environment variable ```H1_DEFAULT_OUTPUT``` also.
-
-### Examples
-
-#### View all virtual machines in table format
+###### View all disks in tsv format
 
 ```bash
-{{scope}} vm list -o table
+h1 disk list -o tsv
```

Comment:
should't be {{scope}} here?

Reply:
No, this file is output of generator, so I fix there bug that the scope in ```docs/common-arguments.md``` (there is custom logic for that file) are not resolved. In ```docs/*``` we store final, scoped content.

Reply:
https://github.com/hyperonecom/h1-cli/pull/167/files#diff-b4396a49c757347917b1429209911d5cR17